File: crates/comment-lint-core/tests/fixtures/go/superfluous.go
package superfluous

// increment counter

func incrementCounter(counter int) int {
	counter++
	return counter
}

// get user name

func getUserName(user string, name string) string {
	return user + name
}

// validate input data

func validateInputData(input string, data []byte) error {
	if len(data) == 0 {
		return nil
	}
	return nil
}

// calculate total price

func calculateTotalPrice(total float64, price float64) float64 {
	return total + price
}

File: crates/comment-lint-core/tests/fixtures/rust/superfluous.rs
// get user name
fn get_user_name(user: &str) -> String {
    user.to_string()
}

// increment counter value
fn increment_counter_value(counter: &mut i32, value: i32) {
    *counter += value;
}

// validate input data
fn validate_input_data(input: &str, data: &[u8]) -> bool {
    !input.is_empty() && !data.is_empty()
}

// calculate total price
fn calculate_total_price(total: f64, price: f64) -> f64 {
    total + price
}
